packages/commons-ui-core/src/QuickLinks/QuickLinks.js

```diff
@@ -0,0 +1,58 @@
+import { Box, Link, Typography } from "@mui/material";
+import React from "react";
+
+const QuickLinks = React.forwardRef(function QuickLinks(props, ref) {
+  const { linkComponent, links, LinksProps, LinkProps, title, TitleProps, sx } =
+    props;
+  const LinkComponent = linkComponent || Link;
```

Comment:
Since you have `LinksProps`, shouldn't `linkComponent` be `LinksProps?.component`?